src/main/java/uk/gov/hmcts/juror/api/moj/report/bespoke/JurorAmendmentByDateReport.java

```diff
@@ -0,0 +1,88 @@
+package uk.gov.hmcts.juror.api.moj.report.bespoke;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import org.springframework.transaction.annotation.Transactional;
+import uk.gov.hmcts.juror.api.moj.controller.reports.request.StandardReportRequest;
+import uk.gov.hmcts.juror.api.moj.controller.reports.response.AbstractReportResponse;
+import uk.gov.hmcts.juror.api.moj.domain.Role;
+import uk.gov.hmcts.juror.api.moj.report.AbstractReport;
+import uk.gov.hmcts.juror.api.moj.service.JurorServiceMod;
+import uk.gov.hmcts.juror.api.moj.service.UserService;
+import uk.gov.hmcts.juror.api.moj.service.audit.JurorAuditService;
+import uk.gov.hmcts.juror.api.moj.utils.SecurityUtil;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+
+@Component
+public class JurorAmendmentByDateReport extends AbstractJurorAmendmentReport {
+
+    @Autowired
+    public JurorAmendmentByDateReport(
+        JurorAuditService jurorAuditService,
+        JurorServiceMod jurorService,
+        UserService userService) {
+        super(jurorAuditService, jurorService, userService);
+    }
+
+    @Override
+    @Transactional(readOnly = true)
+    public JurorAmendmentReportResponse getStandardReportResponse(
+        StandardReportRequest request) {
+        JurorAmendmentReportResponse response = new JurorAmendmentReportResponse();
+
```

Comment:
could add some logging here to indicate the start of the request, e.g.        log.info("Fetching Juror amendment report for juror: {}  by User {}......

The one at the end to say when its done       log.info("Fetched Juror amendment report for juror: {}  by User {}......

might help debug issues